extensions/cms/src/cmsResource/cmsResourceCommands.ts

```diff
@@ -0,0 +1,185 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+import * as nls from 'vscode-nls';
+import * as azdata from 'azdata';
+import { AppContext } from '../appContext';
+import { TreeNode } from './treeNode';
+import { CmsResourceTreeProvider } from './tree/treeProvider';
+import { CmsResourceEmptyTreeNode } from './tree/cmsResourceEmptyTreeNode';
+import { RegisteredServerTreeNode } from './tree/registeredServerTreeNode';
+import { ServerGroupTreeNode } from './tree/serverGroupTreeNode';
+import { CmsResourceTreeNode } from './tree/cmsResourceTreeNode';
+
+const localize = nls.loadMessageBundle();
+
+export function registerCMSServerCommand(appContext: AppContext, tree: CmsResourceTreeProvider): void {
```

Comment:
looks like  there is inconsistent `CMS` vs. `Cms` casing between objects and methods.   Let's be consistent.  I'd personally prefer `Cms`.